backend/src/schema/resolvers/registration.spec.js

```diff
@@ -340,6 +340,7 @@ describe('SignupVerification', () => {
       ) {
         id
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-09-12T15:27:41Z" title="Thursday, September 12th 2019, 5:27:41 pm +02:00">Sep 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -352,6 +353,7 @@ describe('SignupVerification', () => {
         password: '123',
         email: 'john@example.org',
         termsAndConditionsAgreedVersion: '0.0.1',
```


```suggestion
        termsAndConditionsAgreedVersion: null,
```

As I said in our pair-programming, it's confusing to see this. Could we simulate a user that has never agreed to terms and conditions yet? Because it's not possible to have a `termsAndConditionsAgreedVersion` set but a missing `termsAndConditionsAgreedAt`.